package.json

```diff
@@ -82,7 +82,9 @@
     "**/node-sass": "^8.0.0",
     "**/trim": "0.0.3",
     "word-wrap": "1.2.4",
-    "electron-builder/app-builder-lib/@electron/universal/dir-compare/minimatch": "^3.0.5"
+    "electron-builder/app-builder-lib/@electron/universal/dir-compare/minimatch": "^3.0.5",
+    "**/semver": "^7.5.2",
+    "rawproto/protobufjs": "^7.2.4"
```

Comment:
Is this version compatible with "rawproto": "^0.7.6"?

Reply:
@GnaneshKunal

Reply:
Looking into this. Basic RI UI test did pass. Running tests/same code from the library

Reply:
Ran rawproto tests. Worked fine for protobuf `^7.2.4`